packages/project-account-switcher/src/presenters/ProjectAccountSwitcherPresenter.test.js

```diff
@@ -0,0 +1,42 @@
+import React from "react";
+import renderer from "react-test-renderer";
+import ProjectAccountSwitcherPresenter from "./ProjectAccountSwitcherPresenter";
+
+describe("project-account-switcher/presenters/ProjectAccountSwitcherPresenter", () => {
+  [
+    {
+      description: "renders without props",
+      props: {}
+    },
+    {
+      description: "renders with all props",
+      props: {
+        accountTitle: "Accounts",
+        accounts: [
+          { id: "1", label: "Account 1" },
+          { id: "2", label: "Account 2" }
+        ],
+        activeAccountId: "1",
+        activeProjectId: "2",
+        onAccountClick: function onAccountClick() {},
+        onProjectClick: function onProjectClick() {},
+        projectTitle: "Projects",
+        projects: [
+          { id: "1", label: "Project 1" },
+          { id: "2", label: "Project 2" }
+        ]
+      }
+    }
+  ].forEach(({ description, props: { children, ...otherProps } }) => {
+    it(description, () => {
```

Comment:
Not a big deal but @morrisallison created a helper to streamline this.  See here: https://github.com/Autodesk/hig/pull/1092/files#r208754940

Reply:
👍 addressed in https://github.com/Autodesk/hig/pull/1102/commits/551f97e14209a981a7e7926b17c3bf797c6b93a3